### DISCUSSION TITLE
Possible to train it on my own voice?

### DISCUSSION THREAD

**Discussion by tjthejuggler** [RHVoice/RHVoice#811] [General]
If I make recordings of myself talking, am I able to do this with my own voice?

**Comment by zstanecic** [comment]
yes, it is possible, but think about it twice. For example, is your voice good for speech synthesis?

**Reply by tjthejuggler** [reply]
Thanks so much for the response. I honestly have no idea if my voice is good for speech synthesis. I'm curious to find out though. Do you have any tips or advice on how to get started? What form I should get the audio in, etc.
